test/jdk/jdk/incubator/vector/Byte256VectorLoadStoreTests.java

```diff
@@ -479,8 +479,8 @@ static void loadStoreMemorySegment(IntFunction<byte[]> fa,
 
     @Test(dataProvider = "byteByteProviderForIOOBE")
     static void loadMemorySegmentIOOBE(IntFunction<byte[]> fa, IntFunction<Integer> fi) {
-        MemorySegment a = toSegment(fa.apply(SPECIES.length()), i -> MemorySegment.allocateNative(i, Byte.SIZE, MemorySession.openImplicit()));
-        MemorySegment r = MemorySegment.allocateNative(a.byteSize(), Byte.SIZE, MemorySession.openImplicit());
+        MemorySegment a = toSegment(fa.apply(SPECIES.length()), i -> MemorySession.openImplicit().allocate(i, Byte.SIZE));
```

Comment:
This file also seems like it could  benefit from using `MemorySegment.allocateNative`